tests_21/conftest.py

```diff
@@ -0,0 +1,55 @@
+"""Homework 21 Test case - conf test"""
+
+import pytest
+from homework_12 import Bank, Reader, Book
+
+# def pytest_addoption(parser):
+#     """"
+#      pytest_addoption func
+#     """
+#     parser.addoption(
+#         "--debug", action="store_true", help="Enable debugging output"
+#     )
```

Comment:
Why did you commented this code? This approach  didn’t work for you to setup log level?